content/en/blog/2022-07-05-unikraft-gsoc-intel-sgx.md

```diff
@@ -0,0 +1,32 @@
++++
+title = "Improving security and isolation of Unikraft with Intel SGX "
+date = "2022-06-13T00:00:00+01:00"
+authors = [
+  "Xiangyi Meng"
+]
+tags = ["Security", "Intel SGX"]
++++
+Trusted execution environment (TEE), especially Intel SGX, is a popular and powerful tool to provide hardware-based isolation for highly sensitive code and data. Today’s commercial clouds (Microsoft Azure DCsv2, DCsv3/DCdsv3 series, IBM Cloud Bare Metal z15 series, and Alibaba Cloud Bare Metal Instance) more or less have provided their support to Intel SGX. As a powerful and cloud-ready unikernel development kit, Unikraft should enable this feature to give users an option to protect their sensitive code and data.
```

Comment:
Refer Unikraft's interest in security by linking this: https://github.com/unikraft/docs/blob/main/content/en/docs/features/security.md